docs/testing-deployment-publishing/branching-collaboration/branching.md

```diff
@@ -0,0 +1,233 @@
+
+:::note Few things to note here
+
+- At the moment, FlutterFlow only supports merging into `main`.
+- Only the user who initiated the merge can access both the `main`` branch and the branch during an ongoing merge.
+- Changes that don't conflict are resolved automatically.
+- You cannot current undo a merge, but a merge can be aborted if you want to start over.
+- If you leave the project during the merging and come back, the progress you have made on the merge will be preserved.
+
+:::
+
+Let's see how to create, merge branches, and resolve conflicts with a practical example. Here's how you do it.
+
+1. Create a new branch from the ‘main’ branch.
+
+:::tip
+
+You can also create a new branch from any existing branch.
+
+:::
+
+<div style={{
+    position: 'relative',
+    paddingBottom: 'calc(56.67989417989418% + 41px)', // Keeps the aspect ratio and additional padding
+    height: 0,
+    width: '100%'
+}}>
+    <iframe 
+        src="https://demo.arcade.software/5n61rPZR7WuWxs0lTFkE?embed&show_copy_link=true"
+        title="Sharing a Project with a User"
+        style={{
+            position: 'absolute',
+            top: 0,
+            left: 0,
+            width: '100%',
+            height: '100%',
+            colorScheme: 'light'
+        }}
+        frameborder="0"
+        loading="lazy"
+        webkitAllowFullScreen
+        mozAllowFullScreen
+        allowFullScreen
+        allow="clipboard-write">
+    </iframe>
+</div>
+
+2. Make some changes in the new branch, and from the same menu, select the **Merge** option.
+
+:::info
+
+- You could also select the **Rebase** option, which rewrites the feature branch's history to appear as if it was built directly on top of the latest 'main' branch. However, It's **important to note** that if you're working in a collaborative setting where maintaining an accurate historical record is important, it's safer to simply use the **Merge** option.
+- Know more about Merge vs Rebase [**here**](#merge-vs-rebase).
+
+:::
+
+<div style={{
+    position: 'relative',
+    paddingBottom: 'calc(56.67989417989418% + 41px)', // Keeps the aspect ratio and additional padding
+    height: 0,
+    width: '100%'
+}}>
+    <iframe 
+        src="https://demo.arcade.software/VHuZbhTfT8Q8GbaMzMWA?embed&show_copy_link=true"
+        title="Sharing a Project with a User"
+        style={{
+            position: 'absolute',
+            top: 0,
+            left: 0,
+            width: '100%',
+            height: '100%',
+            colorScheme: 'light'
+        }}
+        frameborder="0"
+        loading="lazy"
+        webkitAllowFullScreen
+        mozAllowFullScreen
+        allowFullScreen
+        allow="clipboard-write">
+    </iframe>
+</div>
+
```

Comment:
Add p tag after this to create some space

Reply:
Added!